client/verta/verta/registry/check_model_dependencies.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+
+from typing import Set, Type
+import warnings
+
+from verta.registry import VertaModelBase
+from verta.environment import Python
+from verta._internal_utils import model_dependencies as md
+
+
+def check_model_dependencies(
+        model: Type[VertaModelBase],
+        environment: Python,
+        raise_for_missing: bool = False,
+        ) -> bool:
+    """
+    Attempt to scan the provided model class for dependencies and verify that
+    the provided environment includes the necessary packages to support them.
+
+    .. version_added:: 0.22.1
```

Comment:
`0.22.1` is going out first thing next week alongside our platform's `2023_03` release, so this probably shouldn't be considered user-ready until the following version.

```suggestion
    .. versionadded:: 0.22.2
```

Reply:
On that note, if we're looking to merge this today/Monday, we should hide this function at least from our API documentation by either

1. making it private by renaming the function to `_check_model_dependencies()`
2. commenting out the docstring

Reply:
Fixed, and I'm assuming we will not merge this before EOD, so we can take the necessary steps to hide it next week.